src/arch/svm/SpokeUtils.ts

```diff
@@ -848,21 +855,24 @@ export async function getAssociatedTokenAddress(
   return associatedToken;
 }
 
-export function getRelayDataHash(relayData: RelayData, destinationChainId: number): string {
+export function getRelayDataHash(relayData: RelayDataWithMessageHash, destinationChainId: number): string {
   assert(relayData.message.startsWith("0x"), "Message must be a hex string");
   const uint64Encoder = getU64Encoder();
 
   const svmRelayData = toSvmRelayData(relayData);
   const relayDataEncoder = SvmSpokeClient.getRelayDataEncoder();
   const encodedRelayData = relayDataEncoder.encode(svmRelayData);
   const encodedMessage = Buffer.from(relayData.message.slice(2), "hex");
```

Comment:
I thought the point of using `RelayDataWithMessageHash` is to no longer require the object being passed in to have `message` defined? I'd actually think it would be better to re-define `RelayDataWithMessageHash` as a `Omit<RelayData, "message"> & { messageHash: string}`. Right now, it seems like if I wanted to call this function, I still need a `message` field in relay data, so there isn't really a point to add a message hash ever.